ArcGISRuntimeSDKQt_CppSamples/Analysis/FindFeaturesUtilityNetwork/FindFeaturesUtilityNetwork.qml

```diff
@@ -0,0 +1,130 @@
+// [WriteFile Name=FindFeaturesUtilityNetwork, Category=Analysis]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import Esri.Samples 1.0
+
+Item {
+
+    // add a mapView component
+    MapView {
+        id: view
+        anchors.fill: parent
+    }
+
+    // Declare the C++ instance which creates the scene etc. and supply the view
+    FindFeaturesUtilityNetworkSample {
+        id: model
+        mapView: view
+    }
+
+    Dialog {
+        modal: true
+        standardButtons: Dialog.Ok | Dialog.Cancel
+        x: Math.round(parent.width - width) / 2
+        y: Math.round(parent.height - height) / 2
+        visible: model.terminalDialogVisisble
+
+        ColumnLayout {
+            Text {
+                text: qsTr("Select the terminal for this junction.")
+                Layout.alignment: Qt.AlignHCenter
+            }
+
+            ComboBox {
+                id: terminalSelection
+                model: ["High", "Low"]
+                Layout.alignment: Qt.AlignHCenter
+            }
+        }
+
+        onAccepted: model.multiTerminalIndex(terminalSelection.currentIndex);
+    }
+
+    Dialog {
+        modal: true
+        standardButtons: Dialog.Ok
+        x: Math.round(parent.width - width) / 2
```

Comment:
is Math.round required?

Reply:
After a quick test, it doesn't seem so. I'll remove them.